testutils/logger.go

```diff
@@ -0,0 +1,51 @@
+package testutils
+
+import (
+	"fmt"
+
+	"gopkg.in/src-d/go-log.v1"
+)
+
+type LoggerMock struct {
+	out []string
+}
+
+func (l *LoggerMock) First() string {
```

Comment:
Perhaps better `Pop`?

Reply:
Or `Deque` better

Reply:
I'm used to using `Pop` for LIFO, taking the last one.
What about [`Next()` as in go queues](https://godoc.org/github.com/foize/go.fifo#Queue.Next)?

Reply:
f6a542276925ad8d8a1d9b21ffe9db34d89b8ad1 thanks